web-server/v0.4/src/pages/Search/index.js

```diff
@@ -297,7 +298,7 @@ class SearchList extends Component {
                       >
                         {mapping[field].map(item => {
                           return (
-                            <Select.Option value={item} label={item}>
+                            <Select.Option value={item} label={item} key={uuid.v4()}>
```

Comment:
Is there a requirement that we need to use the `uuid` package here?  Could we construct a unique key value using a monotonically incrementing counter combined with the value?

Reply:
Awesome, that'd be great! I think just adding a `item` as a key would be perfectly fine as each `item` is unique here